packages/web-components/fast-components/src/index.ts

```diff
@@ -3,14 +3,13 @@ export * from "./badge";
 export * from "./button";
 export * from "./card";
 export * from "./checkbox";
-export * from "./custom-properties";
+export * from "./color";
```

Comment:
you'll need to export the recipe behaviors from "./styles/recipes".

If we don't *need* the color recipes exported right now I would hold off